src/helpers/system_io_helper.jl

```diff
@@ -0,0 +1,11 @@
+"""
+    _mkdir(path::AbstractString)::AbstractString
+
+Create directory if it does not exist.
+"""
+function _mkdir(path::AbstractString)::AbstractString
```

Comment:
there is already a function for this: `[mkpath`](https://docs.julialang.org/en/v1/base/file/#Base.Filesystem.mkpath)

Reply:
fix

Reply:
fix